app/views/layouts/application.html.erb

```diff
@@ -10,7 +10,8 @@
 
   <body>
     <% if signed_in? %>
-      Signed in as: <%= current_user.email %>
+      Signed in as: <%= current_user.name %>
```

Comment:
Shouldn't there be an if statement here which displays noface_image.jpg if current_user.avatar? _isn't_ true?

Right now, your default avatar doesn't appear if a user registers without uploading a profile picture.